ReleaseHistory.md

```diff
@@ -12,9 +12,11 @@
 - FND => False negative reduction in dynamic phase
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
+- DEP => upgrade dependency versions
 - NEW => new feature 
 
 ## UNRELEASED
+* DEP: Update 'Microsoft.CodeAnalysis.NetAnalyzers' package from 7.0.0 to 7.0.1 to resolve build warnings. [#903](https://github.com/microsoft/binskim/pull/903)
```

Comment:
@schlaman-ms I think there is an issue here: Microsoft.CodeAnalysis.NetAnalyzers should be within backticks, not single quotes. Can you check if it is rendering corectly?

Reply:
Yup, missed that. Not so used to the backtick for markdown. But I won't miss that again though.